designer/server/src/test/scala/pl/touk/nussknacker/ui/api/NodesApiHttpServiceBusinessSpec.scala

```diff
@@ -926,6 +939,43 @@ class NodesApiHttpServiceBusinessSpec
     }
   }
 
+  "TypingResult schemas shouldn't go out of sync with Codecs" in {
```

Comment:
This spec has end-to-end tests. The test below is like a unit test. It refers to sth which is defined in the NodeApiEndpoints class. So, let's create NodesApiEndpointsSpec and move it there. WDYT?